### ISSUE TITLE
Resume to blank activity on android

### ISSUE THREAD

**Issue by dwd31415** [dwd31415/GameSharing#13]
If the phone resumes an cocos2d-x application with GameSharing after being in standby, it shows a blank activity.

**Comment by dadidzf** [comment]
Hi, Thanks for your work of gamesharing !

Have you fixed this feature - Resume to blank activity on android

BR

**Comment by dwd31415** [comment]
No this feature is not fixed yet, the fix is due for v1.3

**Comment by dadidzf** [comment]
Hi, Do you have any idea on this issue, i have used this project in my game developing with cocos2dx.
but i am not good at java & android.

**Comment by dadidzf** [comment]
i have been hanging on this issue for a few days.

**Comment by dadidzf** [comment]
Can you help me ?

**Comment by dadidzf** [comment]
https://play.google.com/store/apps/details?id=com.yongwu.tetris_free

**Comment by dwd31415** [comment]
You should not have this problem with the most recent version of GameSharing, I was not able to reproduce the issue with the current version(v1.3).

**Comment by dadidzf** [comment]
yes, It just occurs sometimes, if we connected to google play service success all  the  time, it will work fine, it happens only when the network is not good, and the game disconnected with the google play service. i see the code have dealed with this case. but it still have possibility to show blank activity, when come back from rating, ads click, sharing, or an phone call.  Maybe it's something due to google play service sdk version (i will try another version), or it is due to cocos2dx activity mechanism. 

Thanks.
